tests/conftest.py

```diff
@@ -40,7 +41,7 @@
 HTTPD_ROOT: Path = TEST_Path
 HTTPD_HOST: str = (
     # TODO test with localhost.localdomain
-    "localhost"  # can be '' - maybe also try '0.0.0.0' to bind all
+    "127.0.0.1"  # can be '' - maybe also try '0.0.0.0' to bind all
```

Comment:
after the full history we had on this -- it might be wise to include some comment lines on why this ended up the better choice then the considered alternatives:
- `localhost` (leads to not valid URI) or 
- `localhost.localdomain` (leads to not working on windows, ie unless a etc/hosts entry is added)